javascript-primitives-and-variables/javascript-primitives-and-variables-quiz-notes.md

```diff
@@ -0,0 +1,59 @@
+# javascript-primitives-and-variables-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the purpose of variables?
+
+- How do you **declare** a variable?
+
+- How do you initialize (**assign a value to**) a variable?
+
+- What characters are allowed in variable names?
+
+- What does it mean to say that variable names are "case sensitive"?
+
+- What is the purpose of a string?
+
+- What is the purpose of a number?
+
+- What is the purpose of a boolean?
+
+- What does the `=` operator mean in JavaScript?
+
+- How do you update the value of a variable?
+
+- What is the difference between `null` and `undefined`?
+
+- Why is it a good habit to include "labels" when you log values to the browser console?
+
+- Give five examples of JavaScript primitives.
```

Comment:
Don't forget the quiz questions! These need to be answered & submitted along with your code.

Reply:
Quiz questions should now be updated. Sorry about that!